src/Language/PureScript/Parser/Declarations.hs

```diff
@@ -126,7 +126,7 @@ parseFixityDeclaration = do
   name <- symbol
   return $ FixityDeclaration fixity name alias
   where
-  aliased = (Left <$> parseQualified (Ident <$> identifier))
+  aliased = P.try (Left <$> parseQualified (Ident <$> identifier))
         <|> (Right <$> parseQualified (ProperName <$> uname))
```

Comment:
Yeah, I guess so! I probably just copied the `Ident` case when adding constructors. We intentionally don't use the `Common` one there as `parseIdent` will accept `Op` or `Ident`.